docs/pywbemcli/commands.rst

```diff
@@ -1748,26 +1700,86 @@ the XML response from the WBEM server
 The ``statistics show`` command displays the current pywbemcli managed
 statistics as a table.
 
-The following example shows the use of ``--timestats`` option in
-the interactive mode to display statistics after the enumerate command
-is executed.
+Using this command only makes sense in interactive mode. In interactive mode,
+the pywbemcli managed statistics is maintained for the entire interactive
+session, and executing commands in the interactive session causes the
```

Comment:
how about "executing commands that communicate with the server"

Reply:
DONE, added that as proposed.